README.md

```diff
@@ -118,6 +118,49 @@ example :
   - your-page-doc-name
 ```
 
+#### Create a release
+
+> Define new version using [semantic versioning](https://semver.org/)
+
+> Create a new local branch `release/X.Y.Z` from master :
+
+ex: `release/0.20.0`
+
+> Update package version in `package.json`
+
+> Check differences between last release and current and fill `CHANGELOG.md` with updates
+
+> Delete the ```##changes``` title at start of the `CHANGELOG.md` if provided
+
+> Add the date and version at start of `CHANGELOG.md` following this convention:
```

Comment:
It would be nice to make one bullet point abut the changelog, and then multiple sub bullet points for subtasks:

```suggestion
- Check differences between last release and current and fill `CHANGELOG.md` with updates
  - Delete the ```##changes``` title at start of the `CHANGELOG.md` if provided
  - Add the date and version at start of `CHANGELOG.md` following this convention:
```
etc.

What do you think?
You can use the preview in GitHub to get an idea of how it will be rendered: https://github.com/ToucanToco/weaverbird/tree/f/readme-release-update#create-a-release